sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -10,8 +10,8 @@
 from pathlib import Path
 from typing import Dict, List, Optional, Union
 
-from azure.ai.ml._restclient.v2023_04_01_preview.models import JobBase
-from azure.ai.ml._restclient.v2023_04_01_preview.models import PipelineJob as RestPipelineJob
+from azure.ai.ml._restclient.v2023_08_01_preview.models import JobBase as JobBase_2308
```

Comment:
NIT: Is it necessary to add the suffix? we have updated all reference for the 2 rest class

Reply:
yes, done.